contracts/src/compute_engines/summary_stats/library.cairo

```diff
@@ -93,7 +93,7 @@ namespace SummaryStats {
         offset: felt,
         tick_arr: TickElem**,
     ) {
-        if (idx == last_idx) {
+        if (idx + offset == last_idx) {
```

Comment:
can we add a test that would have caught this logic bug

Reply:
forgot to commit, one second